src/ape/api/projects.py

```diff
@@ -91,7 +91,23 @@ def cached_manifest(self) -> Optional[PackageManifest]:
         if it exists and is valid.
         """
 
-        return _load_manifest_from_file(self.manifest_cachefile)
+        manifest = _load_manifest_from_file(self.manifest_cachefile)
+        if manifest is not None:
```

Comment:
Hmm, what is the implications of this?

Reply:
What exactly do you mean here? What are you thinking?

The cached manifest is optional because it does not always exist.

Reply:
Add thoughts here: https://github.com/ApeWorX/ape/pull/1291